modules/effects/src/main/scala/dev/profunktor/redis4cats/algebra/scripts.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018-2019 ProfunKtor
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.profunktor.redis4cats.algebra
+
+import dev.profunktor.redis4cats.effects.ScriptOutputType
+
+trait ScriptCommands[F[_], K, V] extends Scripting[F, K, V]
+
+trait Scripting[F[_], K, V] {
+  def eval[R](script: String, returnType: ScriptOutputType[V, R], keys: K*): F[R]
```

Comment:
Nitpick: I would name it `output` instead of `returnType`.

Reply:
Changed!